app/models/main_page_refresher.rb

```diff
@@ -0,0 +1,28 @@
+module MainPageRefresher
+  def prepare_data(transaction)
+    organization = transaction.organization
+    transaction.bank_account.touch
```

Comment:
@shir our model bank_account has `after_touch :recalculate_amount!`, but `touch` work only associated model (`transaction` in current example) change `updated_at`, it's not working after :create or :destroy. I think we can do new task for this bug, because bank_account must recalculate amount after create and destroy of transactions.